lib/stream.js

```diff
@@ -45,37 +45,50 @@ Stream.Stream = Stream;
 try {
   const types = require('util').types;
   if (types && typeof types.isUint8Array === 'function') {
```

Comment:
This needs to be updated to check for the correct function.

Reply:
done.